nais.yaml

```diff
@@ -31,19 +31,11 @@ spec:
     - "{{this}}"
 {{/each}}
   accessPolicy:
-    outbound:
-      rules:
-        - application: digisos
     inbound:
       rules:
-        - application: digisos
```

Comment:
trenger vi `digisos` fortsatt, siden proxy-appen heter det i labs (og labs skal leve litt til)? 

kanskje vi må legge til `sosialhjelp-proxy-dev` og `sosialhjelp-proxy-mock` både som outbound og inbound?

Reply:
Nå ble jeg litt usikker.. Men er det access policies på ting som kjører i `labs-gcp`? Kan uansett beholde den inntil videre da. :)

Mock alt har fungert godt i mock-miljø på `dev-gcp`, uten at jeg har sett noen inbound eller outbound rules definert eksplisitt for den.

Ser at vi har en del outbound rules i selve proxy-appen, men ikke for alle appene våre, og de drar inn en variabel jeg ikke kan se at er definert noe sted. 🤔 

`- host: "sosialhjelp-veiviser{{domene}}"`

Reply:
Update: AccessPolicy gjelder også på labs. Har derfor lagt tilbake accessPolicy for `digisos`, samt hver app som trenger å snakke med mock-alt i `labs-gcp`.